compose/config/config_schema_v1.json

```diff
@@ -103,6 +103,7 @@
         "restart": {"type": "string"},
         "security_opt": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "shm_size": {"type": ["number", "string"]},
+        "sysctls": {"$ref": "#/definitions/list_or_dict"},
```

Comment:
We're not adding anything new to the v1 schema anymore as we want to deprecate it in the future.